docs/intro/overview.rst

```diff
@@ -190,67 +130,38 @@ this is just the surface. Scrapy provides a lot of powerful features for making
 scraping easy and efficient, such as:
 
 * Built-in support for :ref:`selecting and extracting <topics-selectors>` data
-  from HTML and XML sources
+  from HTML/XML sources using CSS selectors extended and XPath expressions,
+  with helper methods to extract using regular expressions.
 
-* Built-in support for cleaning and sanitizing the scraped data using a
-  collection of reusable filters (called :ref:`Item Loaders <topics-loaders>`)
-  shared between all the spiders.
+* An :ref:`interactive shell console <topics-shell>` (IPython aware) for trying
+  out the CSS and XPath expressions to scrape data, very useful when writing or
+  debugging your spiders.
 
 * Built-in support for :ref:`generating feed exports <topics-feed-exports>` in
   multiple formats (JSON, CSV, XML) and storing them in multiple backends (FTP,
   S3, local filesystem)
 
-* A media pipeline for :ref:`automatically downloading images <topics-images>`
-  (or any other media) associated with the scraped items
-
-* Support for :ref:`extending Scrapy <extending-scrapy>` by plugging
-  your own functionality using :ref:`signals <topics-signals>` and a
-  well-defined API (middlewares, :ref:`extensions <topics-extensions>`, and
-  :ref:`pipelines <topics-item-pipeline>`).
-
-* Wide range of built-in middlewares and extensions for:
-
-  * cookies and session handling
-  * HTTP compression
-  * HTTP authentication
-  * HTTP cache
-  * user-agent spoofing
-  * robots.txt
-  * crawl depth restriction
-  * and more
-
 * Robust encoding support and auto-detection, for dealing with foreign,
   non-standard and broken encoding declarations.
 
-* Support for creating spiders based on pre-defined templates, to speed up
-  spider creation and make their code more consistent on large projects. See
-  :command:`genspider` command for more details.
-
-* Extensible :ref:`stats collection <topics-stats>` for multiple spider
-  metrics, useful for monitoring the performance of your spiders and detecting
-  when they get broken
-
-* An :ref:`Interactive shell console <topics-shell>` for trying XPaths, very
-  useful for writing and debugging your spiders
-
-* A :ref:`System service <topics-scrapyd>` designed to ease the deployment and
-  run of your spiders in production.
+* Strong :ref:`extensibility support <extending-scrapy>` and lots of built-in
+  extensions and middlewares to handle things like cookies, crawl throttling,
+  HTTP caching, HTTP compression, user-agent spoofing, robots.txt,
+  stats collection and many more.
```

Comment:
I think we can extract some items from this list to a separate items to make them more visible, e.g. crawl throttling, HTTP caching, "Jobs: pausing and resuming crawls" may deserve their own `*`. It doesn't hurt to have a larger list of Scrapy advantages :) Throttling / caching / pausing are the things which developers can benefit from immediately, they are nice to have out-of-box.